suite-common/formatters/src/makeFormatter.tsx

```diff
@@ -38,9 +40,11 @@ export interface Formatter<TInput, TOutput, TDataContext extends DataContext = D
  */
 export const makeFormatter = <TInput, TOutput, TDataContext extends DataContext = DataContext>(
     format: FormatDefinition<TInput, TOutput, TDataContext>,
+    displayName: string = 'Formatter',
 ): Formatter<TInput, TOutput, TDataContext> => {
     const formatter: Formatter<TInput, TOutput, TDataContext> = props =>
         <>{format(props.value, props)}</> ?? null;
+    formatter.displayName = displayName ?? 'Formatter';
```

Comment:
```suggestion
    formatter.displayName = displayName;
```

since the `displayName` defines to default value `Formatter` in function signature, this check for nullable value is redundant.

Reply:
done